package.json

```diff
@@ -75,7 +75,15 @@
     "extends": "@logux/eslint-config/browser",
     "rules": {
       "no-invalid-this": "off"
-    }
+    },
+    "overrides": [
+      {
+        "files": ["test/index.test.js"],
+        "rules": {
+          "es5/no-arrow-functions": "off"
```

Comment:
Can you move this ignore to test as a comment? Since this ignore should be applied only to the exact line.

Reply:
Yes, I can. I put it here, into configuration because I found no way to disable one particular rule for a single rule. If you think that it will be ok do disable all rules for the line, I'll do that.

Reply:
Try `// eslint-disable-next es5/no-arrow-function`

Reply:
I am not sure about control comment name, but I am sure about the syntax. I used it in Browserslist (sorry, in airport, can't give you exact code).

Reply:
Done.

Reply:
> Try `// eslint-disable-next es5/no-arrow-function`

Thanks for hint. I searched it in docs yesterday for another project but managed to find just after your ask to update the PR.